packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -308,7 +309,8 @@ const wrapTask = async <T>(opts: any, task: Promise<T>): Promise<T> => {
     result => {
       logger.error(message, {
         event: 'task.end',
-        result: serializeError(result),
+        // avoids "TypeError: Converting circular structure to JSON
+        result: util.inspect(result),
```

Comment:
`result` should be an object, not a string, a better fix would be to handle this where the serialization happens.

Let's revert this for now and we'll take care of this in another PR.

Reply:
reverting eats all S3 errors and reports a serialization error instead, is it really what you want?

Reply:
As a work-around, you can manually extract the safe properties (like `message`) or remove the problematic ones.

Reply:
Where do you suggest I do that? here or in the S3 code ?

Reply:
Here is good enough for now, with a link to my comment :slightly_smiling_face:

Reply:
We can remove this, `xo-server` now knows how to handle circular values (14d06fe754b5fa14a00cc6dc0bbaa0f2f1a80039):

```suggestion
        result: serializeError(result),
```